ownCloud File Provider/FileProviderEnumerator.m

```diff
@@ -234,13 +234,33 @@ - (void)provideItemsForEnumerationObserverFromQuery:(OCQuery *)query
 				{
 					if (!observer.didProvideInitialItems)
 					{
-						OCLogDebug(@"##### PROVIDE ITEMS TO %ld --ENUMERATION-- OBSERVER %@ FOR %@: %@", _enumerationObservers.count, observer.enumerationObserver, query.queryPath, query.queryResults);
+						NSArray <OCItem *> *queryResults = query.queryResults;
+
+						OCLogDebug(@"##### PROVIDE ITEMS TO %ld --ENUMERATION-- OBSERVER %@ FOR %@: %@", _enumerationObservers.count, observer.enumerationObserver, query.queryPath, queryResults);
 
 						observer.didProvideInitialItems = YES;
 
-						if (query.queryResults != nil)
+						if (queryResults != nil)
 						{
-							[observer.enumerationObserver didEnumerateItems:query.queryResults];
+//							NSUInteger offset = 0, count = queryResults.count;
```

Comment:
All this commented code is for debugging reasons or is a left-over?

Reply:
It's a left-over from debugging, but may become useful again in the future, so I didn't remove it.